test/appframework_aws/s1/appframework_aws_test.go

```diff
@@ -1724,9 +1723,6 @@ var _ = Describe("s1appfw test", func() {
 			Expect(err).To(Succeed(), fmt.Sprintf("Unable to upload %s apps to S3 test directory for Standalone", appVersion))
 			uploadedApps = append(uploadedApps, uploadedFiles...)
 
-			// Wait for Standalone to be in READY status
-			testenv.StandaloneReady(ctx, deployment, deployment.GetName(), standalone, testcaseEnvInst)
```

Comment:
just checking why this line was removed?

Reply:
I think @tpereirasplunk made this change to fix failing S1 test case